ngrinder-frontend/src/js/components/perftest/detail/Detail.vue

```diff
@@ -312,7 +317,7 @@
                             } else {
                                 this.$router.push(`/perftest/${res.data}`);
                             }
-                        }).catch(error => console.log(error));
+                        }).catch(() => this.showErrorMsg(this.i18n('perfTest.message.save.error')));
```

Comment:
about removing it? ```() => ```

Simply print the message.
Can not you just call a function?
```
}).catch(() => this.showErrorMsg(this.i18n('perfTest.message.save.error')));
=>
}).catch(this.showErrorMsg(this.i18n('perfTest.message.save.error')));
```

check plz.

Reply:
I think if you come into the catch,
Should not the state value be turned back?

``` this.test.status = 'SAVED'; ```